aspnetcore-features/MapControllersVsMapControllerRouteSln/MapControllersVsMapControllerRoute/Controllers/HomeController.cs

```diff
@@ -0,0 +1,31 @@
+using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using MapControllersVsMapControllerRoute.Models;
+
+namespace MapControllersVsMapControllerRoute.Controllers;
+
+public class HomeController : Controller
```

Comment:
Don't use a view controller. It pollutes the PR with unnecessary files which are not necessary for the article itself. If we would use a similar one then the CustomersController or even re-use (if possible) that would be the best option.

Reply:
The HomeController as a View controller will better convey the MapControllerRoute usage in the article. All the view files have been removed except for the Index.cshtml which is needed for the demonstration of the MapControllerRoute approach.